packages/react-native/src/integrations/console.ts

```diff
@@ -34,17 +34,26 @@ export type LogType = "log" | "error" | "warn";
 
 export const handleLog = (type: LogType, message: string) => {
   const parsedMessage = JSON.parse(message);
+  const webviewMark = "(WebView) ";
+  const webviewMarkedElement =
+    typeof parsedMessage[0] === "string"
+      ? [webviewMark + parsedMessage[0]]
+      : [webviewMark, parsedMessage[0]];
+  const webviewMarkedMessage = [
+    ...webviewMarkedElement,
+    ...parsedMessage.slice(1),
+  ];
```

Comment:
```suggestion
  const [formatMessage, ...parsedMessage] = JSON.parse(message);
  const webviewMark = "(WebView) ";

  const webviewMarkedMessage =
    typeof formatMessage === "string"
      ? [webviewMark + formatMessage, ...parsedMessage]
      : [webviewMark, formatMessage, ...parsedMessage];
```

I think this looks better. What do you think?